lib/y2lan_restart_common.pm

```diff
@@ -417,7 +417,7 @@ sub open_yast2_lan {
       = ($options{ui} =~ /ncurses/)
       ? "--" . $options{ui}
       : "";
-    $y2_opts = "--ncurses" if is_pvm_hmc();
+    $y2_opts = "--ncurses" if (is_pvm_hmc() or is_spvm());
```

Comment:
we usually use `||` instead of `or` for the mess that involve to use it as it has lower precedence, probably fine for this case but we could change it in case this condition would grow somehow and commas would added on the right...seems to have issues with that i particular (but not expert on this :) )

Reply:
Ahh... I'm starting to remember ..hehe, done